docs/guides/layers.md

```diff
@@ -28,6 +28,30 @@ currently supported by napari, check out the
 introduction, check out the
 [Layer list section in the napari viewer tutorial](layer-list).
 
+(layer_mode)=
+## Layer mode
+
+All our layers support a `mode` property that changes the way you interact
+with the layer from the viewer. These modes are accessible via the layer controls
+widget top buttons and via keyboard shortcuts (these shortcuts will vary depending on the layer type). Currently, there are two base modes:
+
+* Pan and zoom
+![image: Pan/zoom](../images/pan-zoom-tool.png)
```

Comment:
Could we use the icon that's in 0.4.19, the pan icon?
https://github.com/napari/napari/blob/main/napari/resources/icons/pan_arrows.svg
It's confusing that the mode is called `pan_zoom`, but zoom essentially always works with the mouse wheel, so it's really `pan` that's unique to the mode.